### PR TITLE
No exception when service shutting down by ctrl+C

### PR DESCRIPTION
**Pull request by bocus**

### CODE REVIEW

**Comment by bocus** [comment]
test faild

**Comment by dennisroche** [comment]
No need to close the PR if the tests failed. Let's work on it and get it passing.

**Comment by bocus** [comment]
It should be better PR. HttpListner is failed only in new version of some (which one?) dependent library. Quick fix is bad. I think it should be lambda for stopping due backward compatibility. But for now have no time(

**Comment by bocus** [comment]
It's updated version

File: appveyor.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: 1.3.{build}
+version: 1.4.{build}
```
Comment (dennisroche) [inline appveyor.yml]: 👍 the version bump as we're changing from `net40` to `net452`.

File: TopShelf.Owin/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
 // Version information for an assembly consists of the following four values:
 //
 //      Major Version
 //      Minor Version 
 //      Build Number
 //      Revision
-[assembly: AssemblyVersion("1.2.0.0")]
-[assembly: AssemblyFileVersion("1.2.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.4.0.0")]
```
Comment (dennisroche) [inline TopShelf.Owin/Properties/AssemblyInfo.cs]: These didn't need to change as AppVeyor will update them as part of the build

File: TopShelf.Owin/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -3,13 +3,13 @@
 [assembly: AssemblyTitle("TopShelf.Owin")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyProduct("TopShelf.Owin")]
-[assembly: AssemblyCopyright("Copyright ©  2014")]
```
Comment (dennisroche) [inline TopShelf.Owin/Properties/AssemblyInfo.cs]: Actually, I think we should remove `AssemblyCopyright` as it is licensed under MIT

**Comment by dennisroche** [comment]
Thanks. A few minor comments but happy for this to be merged.

File: TopShelf.Owin/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (bocus) [inline TopShelf.Owin/Properties/AssemblyInfo.cs]: Ok, should i leave standart notation 1.*.*.*?

File: TopShelf.Owin/Properties/AssemblyInfo.cs (hunk 3 above)
Comment (bocus) [inline TopShelf.Owin/Properties/AssemblyInfo.cs]: Ok, i will

**Comment by bocus** [comment]
I also should to change readme for net 452

**Pull request merged**